src/webfaf/stats.py

```diff
@@ -85,14 +86,14 @@ def date_filter(query):
         if not release_sum:
             continue
 
-        percentage = int(release_sum * 100.0 / total)
+        percentage = int(release_sum * 100.0 // total)
```

Comment:
When you use // then we do not need the int()

Reply:
It should stay ```/``` because the first division operator is ```float```.